### PR TITLE
Create 2018-10-22--js-core-dev-team-weekly.md

### PR DESCRIPTION
**Pull request by jacobheun**
- [x] Add video url

### CODE REVIEW

**Comment by diasdavid** [comment]
@jacobheun you should have the permissions back again

File: meeting-notes/2018/2018-10-22--js-core-dev-team-weekly.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+- **Lead:** @alanshaw
+- **Notetaker:** @jacobheun
+- **Attendees:**
+  - @terichadbourne
+  - @vasco-santos
+  - @victorbjelkholm
+  - @hugomrdias
+  - @vmx
+- **Recording:** TBC
```
Comment (diasdavid) [inline meeting-notes/2018/2018-10-22--js-core-dev-team-weekly.md]: Where is the mighty recording?

File: meeting-notes/2018/2018-10-22--js-core-dev-team-weekly.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+```
+    - full tests set went  2m 39s to 2m 14s `-25s`
+  - add init arg to ipfs daemon cmd
+    - start-stop tests go and js 49 s > 38s  (go needs the same optimization)
+    - start-stop tests js only 27s > 18s
+    - full tests 2m 16s > 1m 50s
+  - help with [js.ipfs.io](http://js.ipfs.io) upgrade to gatsby 2
+    - Released [js-ipfsd-ctl v0.39.3](https://github.com/ipfs/js-ipfsd-ctl/releases/tag/v0.39.3)
+- Blocked
+    - couple aegir PRs
```
Comment (diasdavid) [inline meeting-notes/2018/2018-10-22--js-core-dev-team-weekly.md]: @hugomrdias which PRs?

File: meeting-notes/2018/2018-10-22--js-core-dev-team-weekly.md (hunk 1 above)
Comment (jacobheun) [inline meeting-notes/2018/2018-10-22--js-core-dev-team-weekly.md]:
```suggestion
- **Recording:** https://youtu.be/wwJI_P7tH-4
```

Comment (jacobheun) [reply meeting-notes/2018/2018-10-22--js-core-dev-team-weekly.md]: Recording added and the video has been properly ordered in the js core playlist on youtube as well.

**Pull request merged**